### DISCUSSION TITLE
Cassandra and AWS Secrets Manager

### DISCUSSION THREAD

**Discussion by sathanas24** [trinodb/trino#19402] [Q&A]
I'm using Trino with the Cassandra connector, and the Cassandra credentials are stored in AWS Secrets Manager. Is there any way to configure the Cassandra connector to get credentials from AWS?

**Comment by hashhar** [comment]
Not directly.

You can however use AWS SM to export the secrets as env vars and then use https://trino.io/docs/current/security/secrets.html
